### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abebeos**
Add some more info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
-tessellation
-===========
+# tessellation
+
 
 ![build](https://img.shields.io/github/actions/workflow/status/Constellation-Labs/tessellation/release.yml?label=build)
 ![version](https://img.shields.io/github/v/release/Constellation-Labs/tessellation?sort=semver)
 
-## Running L0 & L1 in Kubernetes
+The Constellation Network Node Software, written in Scala, ready for Kybernetes Deployment.
```
Comment (sdavidp) [inline README.md:7]:
```suggestion
The Constellation Network Node Software, written in Scala, ready for Kubernetes Deployment.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+
+## Documentation
+
+You'll find thorough resources on the documentation site https://docs.constellationnetwork.io.
```
Comment (sdavidp) [inline README.md:11]:
```suggestion
Thorough resources are available on the [documentation site](https://docs.constellationnetwork.io).

File: README.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+
+Ensure to go through the [Constellation Network Primer](https://docs.constellationnetwork.io/learn) to review the main characteristics of Constellation.
```
Comment (sdavidp) [inline README.md:13]:
```suggestion
The [Constellation Network Primer](https://docs.constellationnetwork.io/learn) provides an overview of the Constellation Network.
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
+
+## Dev Setup / Contributing
+
+* [Dev System Setup](SETUP.md)
+* [Contributing Instructions and Guidelines](CONTRIBUTING.md)
+
+## Quick Start
+
+Running L0 & L1 in Kubernetes
```
Comment (sdavidp) [inline README.md:22]: References to `L0` and `L1` may be ambiguous. Are you referring to the _Global_ L0 and L1, or _Metagraph_ ?

Comment (abebeos) [reply README.md:22]: This is original text.

**Comment by abebeos** [comment]
(note that i currently work over the ui, so I cannot squash. Maintainers should be able though to squash this via the UI, if the repo is setup this way)

File: README.md (hunk 4 above)
Comment (abebeos) [reply README.md]: squashed & forced pushed, including a small correction for this.

**Review by ryle-goehausen — APPROVED** [review]
LGTM

**Pull request merged**